### PR TITLE
Commit label logos

### PR DESCRIPTION
**Pull request by teolemon**
Commit label logos that are in production but not on the repository.

### CODE REVIEW

File: html/images/lang/en/labels/real-california-milk-90x90.png [hunk 1]
```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr" class="client-nojs">
+<head>
+<title>File:Real-california-milk-90x90.png - Open Food Facts EN</title>
```
Comment (hangy) [inline html/images/lang/en/labels/real-california-milk-90x90.png:4]: This seems to be HTML from Wikipedia.

Comment (teolemon) [reply html/images/lang/en/labels/real-california-milk-90x90.png]: fixed. the proper logo was already commited.

**Comment by teolemon** [comment]
If I move them from fr to en, it will break display on fr.off.org
see: https://github.com/openfoodfacts/openfoodfacts-server/issues/3821

**Comment by hangy** [comment]
At least for these two logos, the English version should be used for other languages.

**Comment by teolemon** [comment]
@hangy
- I added FSC to labels/en/
- Max Havelaar, there's already a SVG and PNG copy in labels/en/

**Pull request merged**